maps/metaclub.dmm

```diff
@@ -5998,8 +5998,8 @@
 "clr" = (/turf/simulated/floor{dir = 1; icon_state = "whitepurple"},/area/science/lab)
 "cls" = (/obj/structure/bed/chair/office/light{dir = 1; pixel_y = 3},/turf/simulated/floor{dir = 1; icon_state = "whitepurple"},/area/science/lab)
 "clt" = (/obj/machinery/door_control{dir = 2; id_tag = "rndshuttersup"; name = "Shutters Control Button"; pixel_x = 26; pixel_y = 6},/obj/structure/disposalpipe/trunk,/obj/machinery/disposal{pixel_x = 0},/turf/simulated/floor{dir = 5; icon_state = "whitepurple"; tag = "icon-whitehall (WEST)"},/area/science/lab)
-"clu" = (/obj/machinery/door/firedoor/border_only,/obj/structure/disposalpipe/segment,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/Doorresearch.dmi'; id_tag = ""; name = "Research Break Room"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/turf/simulated/floor{tag = "icon-cafeteria (NORTHEAST)"; icon_state = "cafeteria"; dir = 5},/area/science/hallway{name = "Research Division"})
-"clv" = (/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/Doorresearch.dmi'; id_tag = ""; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
+"clu" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/structure/disposalpipe/segment,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/doorresearch.dmi'; id_tag = "ResearchFoyer"; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
+"clv" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/doorresearch.dmi'; id_tag = ""; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
```

Comment:
Why did smartgit change these files?  PJB says its fine but I dont' understand it and that makes me nervous.